### DISCUSSION TITLE
GitPython with ssh keys

### DISCUSSION THREAD

**Discussion by jvsrvcs** [gitpython-developers/GitPython#1176] [Q&A] [answered]
Using GitPython with ssh keys does not seem possible.  There is a small snippet of documentation provided, but it does not work.

I simply want to do a git pull, using a ssh key.  Please do not jump on the keyboard and reply unless you have actually coded in GitPython doing a git pull with ssh keys.  Just don't do it.

**Comment by Byron** [comment, marked as answer]
It's obvious you are upset of something not working as you would expect it, yet I believe that's never a reason to leave destructive issues like this.

As there is nothing to go on to help improving the documentation or figuring out why whatever you tried isn't working, I am closing this issue.

Feel free to add the information needed to turn this one around in the comments.
